### ISSUE TITLE
Flashing color based on change from previous value

### ISSUE THREAD

**Issue by RockyLogic** [ag-grid/ag-grid#6347]
Is it possible to flash a cell either red or green instead of the default colour based on the negative or positive change of the cell?

Currently, updating rowData with API's setRowData.

Ag Grid - React.js

**Comment by StephenCooper** [comment]
Yes this example shows how to achieve this and how to change the colours used for negative or positive change. 

https://www.ag-grid.com/react-data-grid/flashing-cells/#example-flashing-delay-changes

**Comment by RockyLogic** [comment]
Hey Stephen Thanks for the reply, wanted to follow up but at which point within ag grid would it be able to access previous cell value and new cell value, and be able to change the CSS property without the flash being triggered already?

**Comment by StephenCooper** [comment]
You should be able to follow the same pattern and use the refresh method as we do for these in built renderers

https://github.com/ag-grid/ag-grid/blob/latest/grid-community-modules/core/src/ts/rendering/cellRenderers/animateShowChangeCellRenderer.ts

**Comment by RockyLogic** [comment]
Thanks Stephan this would work perfectly
